### PR TITLE
Todolist review

### PR DESCRIPTION
**Pull request by arslanbisharat**
This is the 4th project of the Main Javascript curriculum at [Microverse](https://www.microverse.org/) - @microverseinc

* The objective is to build a page with interactivity using everything learn about **Objects**, **Factory Functions**, The **Module Pattern**
* The project was completed using Javascript language.

>**note** I completed this project alone

### CODE REVIEW

**Comment by arslanbisharat** [comment]
### note
Thank you for  reviewing my project i kept default value high

File: package.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
+{
+  "name": "to-do-list",
+  "version": "1.0.0",
+  "description": "This is To do list",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```
Comment (Torres-ssf) [inline package.json]: - [ ] We need to use at least 2 scripts here, one for development and another one for production. Each one should inform in each mode Webpack should run (development and production).

**Review by Torres-ssf — CHANGES_REQUESTED** [review]
### Changes required :muscle: 
Hi there @arslanbisharat 
How are you?
Nice job so far :clap: :clap: 
We only need to make some changes before moving on :smile: 

- [x] The stickler file was added but it seems you forgot to set up the linter here on Github. We need to fix that and allow the linter to verify your repo in the PR. Check the instructions [here](https://github.com/microverseinc/linters-config/tree/master/javascript) 

- [x] Your eslint file is different from the one [provided](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json) from Microverse. We need to fix that.

- [x] I noticed that your package.json file has no scripts defined. We need to add at least 2 scripts, development, and production and properly inform how Webpack should run in each one of them. Check my comment below.

- [x] The README file needs to have information on how the user can download and test your app locally. This instruction is related to the scripts available at the package file. 

- [x]  The user should be able to edit the details of a to-do item, like title and description.

Please kindly submit for code review again after making these changes.

Feel free to message me on slack if you have any questions :smile: 
Happy Coding :computer: 
**Sergio Torres**
[Twitter](https://twitter.com/torres_ssf) [LinkedIn](https://www.linkedin.com/in/torres-ssf/) [Portifolio](https://www.torres-ssf.com/)

**Review by fatymahmed — CHANGES_REQUESTED** [review]
## Hi @arslanbisharat 
Great job fixing most of the suggested changes 👏👏, however, a few things need to be fixed
- [ ] Set up linter and make sure it passes the check 
- [ ] The user should be able to edit a `todo's` details, I can't edit the `title` of the `todo` [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/ui_requirement.png)
## Status: Changes Required ♻️

**Comment by arslanbisharat** [comment]
@fatymahmed  @Torres-ssf  I can easily change any information by double clickingon it

**Review by Redvanisation — APPROVED** [review]
# Approved ✔️ 

### Hello team 🌟 

### _Nice work on this project, it seems that you have met the requirements. 👍_ 

### You may merge to master and submit a project completion form. 🚀 

### Good luck! 🍀

**Pull request merged**